Define and activate AWS 9.2.4 release (#234)
giantswarm/releases: release-notes/aws/v9.2.4.md

## :zap: Giant Swarm Release 9.2.4 for AWS :zap:

**If you are upgrading from 9.2.3, upgrading to this release will not roll your nodes. It will only update the apps.**

> This release improves the reliability of NGINX Ingress Controller. Most importantly, kernel and app settings have been tuned to increase out-of-the-box performance, and app Helm chart has been adjusted to improve service availability (minimize disruption) during configuration changes rollout.

Reviewer: ```suggestion
This release improves the reliability of NGINX Ingress Controller. Most importantly, kernel and app settings have been tuned to increase out-of-the-box performance. The app's Helm chart was also adjusted to improve its availability when rolling out configuration changes.
```
